static/js/pomodoro.js

```diff
@@ -149,6 +150,17 @@ function pauseTimer(){
 }
 
 function updatePauseButton(){
+  /*
+    At this point if isReset
+    Is true Then that mean POMODORO is active so we show the pauseButton
```

Comment:
Can you fix the comment format a bit? `At this point if isReset is true ...`

Reply:
```
 /*
    At this point,
    if isReset is true Then that mean POMODORO is active so we show the pauseButton
    else POMODORO is not active so we don't need the pauseButton so we hide it
   */
```

This looks good??